 Network marcusantunius_default  Creating
 Network marcusantunius_default  Created
 Container db  Creating
 Container db  Created
 Container api2  Creating
 Container api1  Creating
 Container api2  Created
 Container api1  Created
 Container nginx  Creating
 Container nginx  Created
Attaching to api1, api2, db, nginx
db     | The files belonging to this database system will be owned by user "postgres".
db     | This user must also own the server process.
db     | 
db     | The database cluster will be initialized with locale "en_US.utf8".
db     | The default database encoding has accordingly been set to "UTF8".
db     | The default text search configuration will be set to "english".
db     | 
db     | Data page checksums are disabled.
db     | 
db     | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db     | creating subdirectories ... ok
db     | selecting dynamic shared memory implementation ... posix
db     | selecting default max_connections ... 100
db     | selecting default shared_buffers ... 128MB
db     | selecting default time zone ... Etc/UTC
db     | creating configuration files ... ok
db     | running bootstrap script ... ok
nginx  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx  | /docker-entrypoint.sh: Configuration complete; ready for start up
db     | performing post-bootstrap initialization ... ok
db     | syncing data to disk ... ok
db     | 
db     | 
db     | Success. You can now start the database server using:
db     | 
db     |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db     | 
db     | initdb: warning: enabling "trust" authentication for local connections
db     | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db     | waiting for server to start....2024-02-17 16:06:55.096 UTC [48] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db     | 2024-02-17 16:06:55.096 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db     | 2024-02-17 16:06:55.097 UTC [51] LOG:  database system was shut down at 2024-02-17 16:06:53 UTC
db     | 2024-02-17 16:06:55.100 UTC [48] LOG:  database system is ready to accept connections
db     |  done
db     | server started
db     | CREATE DATABASE
db     | 
db     | 
db     | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/migration.sql
db     | SET
db     | CREATE TABLE
db     | INSERT 0 5
db     | CREATE TABLE
db     | CREATE INDEX
db     | ALTER TABLE
db     | ALTER TABLE
db     | 
db     | 
db     | waiting for server to shut down...2024-02-17 16:06:55.339 UTC [48] LOG:  received fast shutdown request
db     | .2024-02-17 16:06:55.339 UTC [48] LOG:  aborting any active transactions
db     | 2024-02-17 16:06:55.341 UTC [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
db     | 2024-02-17 16:06:55.341 UTC [49] LOG:  shutting down
db     | 2024-02-17 16:06:55.341 UTC [49] LOG:  checkpoint starting: shutdown immediate
db     | 2024-02-17 16:06:55.346 UTC [49] LOG:  checkpoint complete: wrote 939 buffers (3.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.006 s, sync=0.001 s, total=0.006 s; sync files=0, longest=0.000 s, average=0.000 s; distance=4289 kB, estimate=4289 kB; lsn=0/191B1D0, redo lsn=0/191B1D0
db     | 2024-02-17 16:06:55.351 UTC [48] LOG:  database system is shut down
db     |  done
db     | server stopped
db     | 
db     | PostgreSQL init process complete; ready for start up.
db     | 
db     | 2024-02-17 16:06:55.456 UTC [1] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db     | 2024-02-17 16:06:55.456 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db     | 2024-02-17 16:06:55.456 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db     | 2024-02-17 16:06:55.456 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db     | 2024-02-17 16:06:55.458 UTC [66] LOG:  database system was shut down at 2024-02-17 16:06:55 UTC
db     | 2024-02-17 16:06:55.461 UTC [1] LOG:  database system is ready to accept connections
nginx  | 2024/02/17 16:06:56 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.48.3:81/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/02/17 16:06:56 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.48.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://192.168.48.4:82/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/02/17 16:06:58 [error] 29#29: *4 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/02/17 16:07:00 [error] 29#29: *5 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/02/17 16:07:02 [error] 29#29: *6 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/02/17 16:07:04 [error] 29#29: *7 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx  | 2024/02/17 16:07:06 [error] 29#29: *8 no live upstreams while connecting to upstream, client: 192.168.48.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
db     | 2024-02-17 16:09:24.547 UTC [78] ERROR:  new row for relation "cliente" violates check constraint "cliente_check"
db     | 2024-02-17 16:09:24.547 UTC [78] DETAIL:  Failing row contains (1, 100000, -100000).
db     | 2024-02-17 16:09:24.547 UTC [78] STATEMENT:  UPDATE "cliente" SET "saldo" = "saldo" - $1 WHERE "cliente"."id" = $2
nginx exited with code 0
nginx exited with code 0
api1 exited with code 0
api1 exited with code 0
api2 exited with code 0
api2 exited with code 0
db     | 2024-02-17 16:11:23.372 UTC [1] LOG:  received fast shutdown request
db     | 2024-02-17 16:11:23.372 UTC [1] LOG:  aborting any active transactions
db     | 2024-02-17 16:11:23.374 UTC [1] LOG:  background worker "logical replication launcher" (PID 69) exited with exit code 1
db     | 2024-02-17 16:11:23.374 UTC [64] LOG:  shutting down
db     | 2024-02-17 16:11:23.374 UTC [64] LOG:  checkpoint starting: shutdown immediate
db     | 2024-02-17 16:11:23.378 UTC [64] LOG:  checkpoint complete: wrote 691 buffers (2.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.003 s, sync=0.001 s, total=0.004 s; sync files=0, longest=0.000 s, average=0.000 s; distance=2323 kB, estimate=2323 kB; lsn=0/1B601C8, redo lsn=0/1B601C8
db     | 2024-02-17 16:11:23.382 UTC [1] LOG:  database system is shut down
db exited with code 0
